### ISSUE TITLE
Build error when compiling glew

### ISSUE THREAD

**Issue by t-paul** [mxe/mxe#659]
MXE: master branch
Build: make opencsg

This fails at glew with:
```
'i686-w64-mingw32.static-gcc' -W -Wall -Werror -ansi -pedantic './src/glew-test.c' -o '/data/OpenSCAD/mxe/usr/i686-w64-mingw32.static/bin/test-glew.exe' `'i686-w64-mingw32.static-pkg-config' glew --cflags --libs`
/tmp/ccIq16oS.o:glew-test.c:(.text+0x35): undefined reference to `stderr'
/tmp/ccIq16oS.o:glew-test.c:(.text+0x5f): undefined reference to `stdout'
collect2: error: ld returned 1 exit status
```

I think the root cause is the installed `glew.pc` file which is used by `glew.mk` to build the test program.

Installed `glew.pc` with invalid prefix:
```
prefix=/usr
exec_prefix=${prefix}
libdir=/data/OpenSCAD/mxe/usr/i686-w64-mingw32.static/lib
includedir=${prefix}/include

Name: glew
Description: The OpenGL Extension Wrangler library
Version: 1.12.0
Cflags: -DGLEW_STATIC -I${includedir} 
Libs: -L${libdir} -lGLEW
Requires: gl glu
```

which results in:
```
$ '/data/OpenSCAD/mxe/usr/bin/i686-w64-mingw32.static-pkg-config' glew --cflags --libs
-DGLEW_STATIC -I/usr/include -lGLEW -lopengl32 -lglu32
```

gcc is then using /usr/include/stdio.h causing the error.

**Comment by GuilhermeBernal** [comment]
Fixed by 8d18f9e94ec6d91b2dcb19b4939e73e9faf74ea9

**Comment by TimothyGu** [comment]
Yes, sorry for not closing this in time.
